netmf/NetMF/SRAmqp.cs

```diff
@@ -21,112 +21,156 @@ internal partial class SRAmqp
     {
         public static string AmqpHandleExceeded
         {
+#if (MF_FRAMEWORK_VERSION_V4_2 || MF_FRAMEWORK_VERSION_V4_3 || MF_FRAMEWORK_VERSION_V4_4 || NANOFRAMEWORK_V1_0)
```

Comment:
Is this required? This file should be only included for netmf and nano frameworks.

Reply:
You have point it doesn't. Just remove all that and pushed the change.